bin/checkout-documentation

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+const {docPagesDirectory, current, versions} = require('../constants');
+const {getPrefixedVersion} = require('../src/lib/versionHelper');
+const {execSync} = require('child_process');
+
+execSync(`rm -rf ${docPagesDirectory}`);
```

Comment:
I'm still not convinced that this is safe haha, do we really need the `-f` flag?

Reply:
It needs to be cloned to have the latest stable branch. I think it's simpler this way instead of managing already existing (or not) repositories.